misk/src/test/kotlin/misk/hibernate/ReflectionQueryFactoryTest.kt

```diff
@@ -236,6 +272,15 @@ class ReflectionQueryFactoryTest {
 
     @Constraint(path = "release_date", operator = Operator.IS_NULL)
     fun releaseDateIsNull(): OperatorsMovieQuery
+
+    @Select
+    fun listAsNameAndReleaseDate(session: Session): List<NameAndReleaseDate>
+
+    @Select("name")
+    fun uniqueName(session: Session): String?
```

Comment:
Thoughts on validating single-column projections as nullable?

If the return type were `String` you could write Kotlin code like `queryFactory.newQuery<OperatorsMovieQuery>().uniqueName(session).capitalize()` and it would compile fine even though the name could be `null`.

Reply:
ooh, good idea. As these are general-purpose they’ll almost always need to be nullable. Lets enforce that.

Reply:
Fixed.